apisix/plugins/redirect.lua

```diff
@@ -133,7 +134,12 @@ function _M.rewrite(conf, ctx)
         -- TODO： add test case
         -- PR: https://github.com/apache/apisix/pull/1958
         uri = "https://$host$request_uri"
-        ret_code = 301
+        local method_name = ngx.req.get_method()
```

Comment:
It is recommended to put the following link in the comments for easy reading:

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/308